### ISSUE TITLE
The "Fail to save bot" msg: "If this problem persists, please file an issue onGitHub" drk blue on black contrast, hard to see the "onGitHub" link text.

### ISSUE THREAD

**Issue by WashingtonKayaker** [microsoft/BotFramework-Composer#6153]
## Describe the bug

In one sense, this is a minor fit and finish bug, but it is also an [Accessibility issue](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), which makes it worth fixing. 

The "_Fail to save bot_" error msg contains the following text: 

> If this problem persists, please file an issue onGitHub

The "onGitHub" link text is dark blue, and it is on a black background, this contrast makes it impossible for many users to read. 

## Version

1.3.1-nightly.219396.61c944a

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [x] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## Screenshots

![image](https://user-images.githubusercontent.com/52054121/109878012-8cefe480-7c28-11eb-8d5a-cd9255fda1b4.png)
